install/updateVersion_orig.sh

```diff
@@ -137,13 +137,8 @@ function merge_files() {
   # istio.yaml and istio-auth.yaml file contain cluster-wide installations
```

Comment:
Can we just remove this file ? Anything still using it ?

Reply:
[updateVersion.sh](https://github.com/istio/istio/blob/master/install/updateVersion.sh#L21) calls [updateVersion_orig.sh](https://github.com/istio/istio/blob/master/install/updateVersion_orig.sh#L138), which generates the YAMLs in install/kubernetes. I think the intention is to remove it ASAP and use helm templates, however.